man/unison.1.in

```diff
@@ -310,9 +310,23 @@ or
 .Sy SIGTERM
 works the same way as described above and will additionally stop the continuous
 process. To stop only the continuous process and let the last synchronization
-complete normally, send signal
+complete normally, press
+.Sy Ctrl-D
```

Comment:
Is this talking about sending a literal ^D 0x4, which one would do on BSD by ^V^D, or about typing the character coded as EOF to the tty so that the reader gets EOF?

Reply:
I guess that was a question, but maybe I mean "it would be nice if this were clearer, even though I realize being precise is harder to understand for many".    Perhaps "Send an EOF by pressing the key configured for that, typically ^D.   Or, send a literal 0x4.

Reply:
I will try rewording it. Literal ^D and EOF, either will work